recipes/fastjet/0002-Patch-fastjet-contrib-Makefile.in.patch

```diff
@@ -0,0 +1,25 @@
+From 89b2d558dc9cf31e9a8007444b7683532a2591dd Mon Sep 17 00:00:00 2001
+From: Matthew Feickert <matthew.feickert@cern.ch>
+Date: Tue, 20 Feb 2024 17:12:20 -0600
+Subject: [PATCH] Patch fastjet-contrib Makefile.in
+
+---
+ fastjet-contrib/Makefile.in | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/fastjet-contrib/Makefile.in b/fastjet-contrib/Makefile.in
+index a4645fd..220991d 100644
+--- a/fastjet-contrib/Makefile.in
++++ b/fastjet-contrib/Makefile.in
+@@ -54,7 +54,7 @@ fragile-shared: libfastjetcontribfragile.@DYNLIBEXT@
+
+ fragile_SHARED_SRC_LIST=@FRAGILE_SHARED_SRC_LIST@
+ libfastjetcontribfragile.@DYNLIBEXT@: $(fragile_SHARED_SRC_LIST)
+-	$(CXX) @DYNLIBOPT@ -fPIC -DPIC $(CXXFLAGS) `$(FASTJETCONFIG) --cxxflags --libs` $(fragile_SHARED_SRC_LIST) -o libfastjetcontribfragile.@DYNLIBEXT@
++	$(CXX) @DYNLIBOPT@ -fPIC -DPIC $(CXXFLAGS) `$(FASTJETCONFIG) --cxxflags --libs` $(fragile_SHARED_SRC_LIST) $(LDFLAGS) -o libfastjetcontribfragile.@DYNLIBEXT@
```

Comment:
This patches the `fastjet-contrib/Makefile.in` so that one of the required `Make` targets knows about the `LDFLAGS` that were set in the `./configure` step.

This is what we were missing when @xhochy reviewed the first time (https://github.com/conda-forge/staged-recipes/pull/21052#pullrequestreview-1887593818).